extension/custom.css

```diff
@@ -11,3 +11,38 @@ a[href^="github-mac"] {
 .reponav a[href$="/pulse"] {
 	display: none !important;
 }
+
+/* alternate monospace font stack */
+code,
+kbd,
+pre,
+tt,
+.ace_editor.ace-github-light,
+.blame-sha,
+.blob-code-inner,
+.blob-num,
+.branch-name,
+.commit .sha,
+.commit .sha-block,
+.commit-desc pre,
```

Comment:
Probably not necessary, I just searched the css for anywhere the `Consolas, "Liberation Mono", Menlo, Courier, monospace` stack was used, and copy pasted the selector. I can remove the redundant ones if you would like.